src/platform/stm32f1/hal/ticks.h

```diff
@@ -0,0 +1,14 @@
+/*
+ * SPDX-License-Identifier: MIT
+ * SPDX-FileCopyrightText: 2021 Rafael Silva <silvagracarafael@gmail.com>
+ */
+
+#pragma once
+
+#include "hal/ticks.h"
+#include "util/types.h"
+
+struct ticks_hal_t ticks_hal_init();
+u64 ticks_hal_ticks_ms();
+void ticks_hal_delay_ms(u32 ticks);
+void ticks_hal_delay_us(u32 ticks);
```

Comment:
Do we need to export these functions?

Reply:
The driver needs this, yes, creating a hal is maybe not the ideal method to provide this functionallity